change/@fluentui-react-charting-20c8ae70-f2a4-4ef4-89b6-7ca932f7481d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28414&commitId=b7f80cc110e108b3db832149906ce6dcf7264877&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-HorizontalBarChart</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.Basic - Dark Mode.default.chromium.png|1460| Changed|
|react-charting-HorizontalBarChart.Basic.hover.chromium.png|3744| Changed|
|react-charting-HorizontalBarChart.Variant - RTL.default.chromium.png|1877| Changed|
|react-charting-HorizontalBarChart.With_Benchmark.default.chromium.png|758| Changed|
|react-charting-HorizontalBarChart.Basic - RTL.default.chromium.png|4016| Changed|
|react-charting-HorizontalBarChart.With_Benchmark - Dark Mode.default.chromium.png|294| Changed|
|react-charting-HorizontalBarChart.Variant.default.chromium.png|1877| Changed|
|react-charting-HorizontalBarChart.Variant - Dark Mode.default.chromium.png|617| Changed|
|react-charting-HorizontalBarChart.Basic.default.chromium.png|4016| Changed|
|react-charting-HorizontalBarChart.Basic - Dark Mode.hover.chromium.png|1349| Changed|
|react-charting-HorizontalBarChart.With_Benchmark - RTL.default.chromium.png|759| Changed|

</details>


  <details><summary><b>react-charting-MultiStackBarChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-MultiStackBarChart.Basic_PartToWhole - Dark Mode.default.chromium.png|1361| Changed|
|react-charting-MultiStackBarChart.Basic_PartToWhole - RTL.default.chromium.png|3171| Changed|
|react-charting-MultiStackBarChart.With_Placeholders - Dark Mode.default.chromium.png|416| Changed|
|react-charting-MultiStackBarChart.With_Placeholders - RTL.default.chromium.png|818| Changed|
|react-charting-MultiStackBarChart.Basic_Absolute - RTL.default.chromium.png|2361| Changed|
|react-charting-MultiStackBarChart.With_Placeholders.default.chromium.png|818| Changed|
|react-charting-MultiStackBarChart.Basic_PartToWhole.default.chromium.png|3171| Changed|
|react-charting-MultiStackBarChart.Basic_Absolute - Dark Mode.default.chromium.png|1047| Changed|
|react-charting-MultiStackBarChart.Basic_Absolute.default.chromium.png|2285| Changed|

</details>


  <details><summary><b>react-charting-StackedBarChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-StackedBarChart.Basic.default.chromium.png|2314| Changed|
|react-charting-StackedBarChart.Benchmark.default.chromium.png|355| Changed|
|react-charting-StackedBarChart.Benchmark - RTL.default.chromium.png|898| Changed|
|react-charting-StackedBarChart.Benchmark - Dark Mode.default.chromium.png|785| Changed|
|react-charting-StackedBarChart.Basic - Dark Mode.default.chromium.png|982| Changed|
|react-charting-StackedBarChart.Basic - RTL.default.chromium.png|2355| Changed|
|react-charting-StackedBarChart.Multiple - RTL.default.chromium.png|4575| Changed|
|react-charting-StackedBarChart.Multiple - Dark Mode.default.chromium.png|805| Changed|
|react-charting-StackedBarChart.Multiple.default.chromium.png|955| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Expected changes in screenshots.